CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ cmake_minimum_required(VERSION 3.0)
 project(GLFW VERSION 3.3.2 LANGUAGES C)
 
 set(CMAKE_LEGACY_CYGWIN_WIN32 OFF)
+set(SWITCH_LIBNX ON)
```

Comment:
This isn't needed because it's set by the pkgbuild.

Reply:
makes sense, somehow I could compile it without it and forgot to remove it